lockc/src/bpf/lockc.bpf.c

```diff
@@ -517,4 +537,123 @@ int BPF_PROG(open_audit, struct file *file, int ret_prev)
 	return ret;
 }
 
+SEC("uprobe/add_container")
```

Comment:
Would you mind adding an explicit reference to BPFContain here (and elsewhere) as a comment? We are still trying to get our paper published, so stuff like this matters, unfortunately. We can probably remove such a reference once BPFContain has been fully integrated with lockc down the road.

Reply:
OK, sure, I can do that. Is licensing an issue as well? I know there is mismatch (bpfcontain-rs - GPL, lockc - Apache 2.0 for the userspace part).

Reply:
I'm fine to be flexible on the license to the extent that the GPL itself is. I just chose to conservatively go with GPL to avoid license compatibility issues with the kernel.

Reply:
BTW, aya is licensed under MIT/A2.0